src/FakeItEasy.Analyzer.Shared/FakeItEasy.Analyzer.Shared.projitems

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>c4766c64-39e1-4023-a173-615607c40ad5</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>FakeItEasy.Analyzer.Shared</Import_RootNamespace>

```

Comment:
Is this correct? I noticed that in the language specific projects, the namespace we are using is still `FakeItEasy.Analyzer`.